package.json

```diff
@@ -42,6 +59,9 @@
     "uuid": "^3.2.1"
   },
   "scripts": {
-    "test": "grunt default --verbose"
+    "test": "concurrently 'node server.js' 'npm run test:init' -k --prefix-colors 'yellow,green' --success first",
+    "test:loop": "concurrently 'node server.js' 'npm run test:infinite'  -k --prefix-colors 'yellow,green' --success first",
+    "test:init": "mocha --require @babel/register --timeout 10000 test/index.js",
+    "test:infinite": "while npm run test:init; do :; done"
```

Comment:
Are these different test commands documented somewhere?

Reply:
they are not, but you really just have to use `test`, the other commands are just for making the instructions more modular, or in the case of `test:loop`, run `test` it until it fails.